test/unit/contentChunk.js

```diff
@@ -286,16 +286,27 @@ describe('contentChunk: ', () => {
 
     it('should return an array with promise functions to remove the payloads', async () => {
       const td = testUtils.clone(transaction)
+      td.orchestrations = [{ request: {}, response: {} }]
+      td.routes = [{ request: {}, response: {}, orchestrations: [{request: {}, response: {} }] }]
```

Comment:
This is a little bit confusion to follow. Lets construct the object in one go to make it easier to read
```
..
td.orchestrations = [
  { 
    request: {
      bodyId: await testUtils.createGridFSPayload('<HTTP orchestration body request>')
    }, 
    response: {
      bodyId: await testUtils.createGridFSPayload('<HTTP orchestration body request>')
    } 
  }
]
...
```